vss-graphql/src/index.ts

```diff
@@ -1,3 +1,6 @@
+const _ = require('lodash');
+const sqlite3 = require("sqlite3");
```

Comment:
nitpick, but would you please use `import` statements?

as for `lodash`, it's better to import only `set`.